### ISSUE TITLE
[Add new/Edit device] Snomed multi select is missing aria-required

### ISSUE THREAD

**Issue by BobanL** [CDCgov/prime-simplereport#6404]
## Area or Component

Support Admin > Add new device > Snomed code multiselect 
Support Admin > Edit existing device > Snomed code multiselect 

## Quick Summary

Snomed code multiselect should have `aria-required=true`

## Accessibility Violation

https://docs.deque.com/issue-help/1.0.0/en/aria-required-missing

## Action Requested

Add `aria-required` to the multiselect on both device pages

## Source Code (optional)

[which file or line is the issue attached to?]: #

**Comment by johanna-skylight** [comment]
Oh this is also getting fixed by [this PR](https://github.com/CDCgov/prime-simplereport/pull/6438) as the multiselect is a common component
